canonical-web-and-design/jaas-dashboard
# Updates docker yarn run to use --production

## Done

- Updates docker yarn run to use --production

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Check nothing is broken
